src/rust/cryptography-x509-verification/src/policy/mod.rs

```diff
@@ -251,92 +253,122 @@ impl<'a, B: CryptoOps> Policy<'a, B> {
             minimum_rsa_modulus: WEBPKI_MINIMUM_RSA_MODULUS,
             permitted_public_key_algorithms: Arc::clone(&*WEBPKI_PERMITTED_SPKI_ALGORITHMS),
             permitted_signature_algorithms: Arc::clone(&*WEBPKI_PERMITTED_SIGNATURE_ALGORITHMS),
-            ca_extension_policy: ExtensionPolicy {
-                // 5280 4.2.2.1: Authority Information Access
-                authority_information_access: ExtensionValidator::maybe_present(
-                    Criticality::NonCritical,
-                    Some(common::authority_information_access),
+            ca_extension_policy: ExtensionPolicy::new(HashMap::from([
+                (
+                    // 5280 4.2.2.1: Authority Information Access
+                    AUTHORITY_INFORMATION_ACCESS_OID,
+                    ExtensionValidator::maybe_present(
```

Comment:
I would be glad to hear any ideas on improving the compactness of the initialisation code here.

I considered implementing a function that accepts an argument for each of these "default" validators, but that would be extremely bug-prone since it would be order-dependent.

Reply:
(The github preview doesn't really make it clear what I'm talking about, so check out the full file)